### PR TITLE
Grid template 01 updated

### LINKED ISSUE #15: Improve Grid 01 and List 01 templates

### PR DESCRIPTION
**Pull request by javier-fernandez-imatia**

### CODE REVIEW

File: templates/grid/01-grid/README.md
Comment (patricia-martinez-imatia) [inline templates/grid/01-grid/README.md:1]: I think, it shoul be define the module
image

File: templates/grid/01-grid/grid/grid-home/grid-home.component.html [hunk 1]
```diff
@@ -1,61 +1,64 @@
-<o-form editable-detail="no" show-header="no" layout-align="start stretch" class="employees-filters-form" fxFill>
+<div class="bg-level-1" fxFill>
+  <o-form editable-detail="no" show-header="no" layout-align="start stretch" class="employees-filters-form" fxFill>
```
Comment (patricia-martinez-imatia) [inline templates/grid/01-grid/grid/grid-home/grid-home.component.html]:
```suggestion
  <o-form editable-detail="no" show-header="no" layout-align="start stretch" class="employees-filters-form fill-form">
```

File: templates/grid/01-grid/README.md [hunk 2]
```diff
@@ -135,3 +136,5 @@ export class MainRoutingModule { }
 * **Filter builder** https://ontimizeweb.github.io/docs/v15/components/data/filterbuilder/overview
 
 * **Grid component** https://ontimizeweb.github.io/docs/v15/components/data/grid/overview
+
+* **Ontimize SCSS surface classes** https://ontimizeweb.github.io/docs/v15/customize/style-guide/#surfaces
```
Comment (patricia-martinez-imatia) [inline templates/grid/01-grid/README.md:145]:
```suggestion
* **Ontimize SCSS surface classes** https://ontimizeweb.github.io/docs/v15/customize/style-guide/#surfaces
* **Form Css Class** https://ontimizeweb.github.io/docs/v15/components/data/form/overview#css-class
```

**Review by patricia-martinez-imatia — CHANGES_REQUESTED** [review]

File: templates/grid/01-grid/grid/grid-home/grid-home.component.ts [hunk 3]
```diff
@@ -1,23 +1,54 @@
...
   encapsulation: ViewEncapsulation.None
 })
-export class GridHomeComponent {
+export class GridHomeComponent implements OnInit {
 
   @ViewChild('filterBuilder', { static: true })
   filterBuilder: OFilterBuilderComponent;
 
+  @ViewChild('grid', { static: true })
+  grid: OGridComponent;
```
Comment (patricia-martinez-imatia) [inline templates/grid/01-grid/grid/grid-home/grid-home.component.ts]: It isnot necessary

File: templates/grid/01-grid/assets/dummy-data/employee-types.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+  "code": 0,
+  "message": "",
+  "totalQueryRecordsNumber": 69,
```
Comment (patricia-martinez-imatia) [inline templates/grid/01-grid/assets/dummy-data/employee-types.json]:
```suggestion
  "totalQueryRecordsNumber": 7,
```

File: templates/grid/01-grid/assets/dummy-data/employee-types.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      "NAME": "Transportation department",
+      "ID": 6385
+    },
+    {
+      "NAME": "Security",
+      "ID": 6386
+    }
+  ],
+  "sqlTypes": {
+    "NAME": 12
```
Comment (patricia-martinez-imatia) [inline templates/grid/01-grid/assets/dummy-data/employee-types.json]:
```suggestion
    "NAME": 12,
    "ID": 4
```

**Review by patricia-martinez-imatia — CHANGES_REQUESTED** [review]

**Review by patricia-martinez-imatia — APPROVED** [review]

**Review by daniel-grana — APPROVED** [review]

**Pull request merged**